server/utils/configs.js

```diff
@@ -60,6 +35,7 @@ function loadClientConfigs(_config) {
 
   // if not passed bcfg object, create one
   if (!(_config instanceof Config)) config = loadConfig('bpanel', _config);
+  const _logger = config.obj('logger');
```

Comment:
Why two loggers in this file?

Reply:
Well this is an internal reference to `_logger` only used in this function. Not technically private but I wanted a way to differentiate. The logger needs to be pulled out of the config so that it's easier to use (you can't just do `config.logger.info()` because of the way bcfg works). 

As far as the two loggers, the one declared/imported at the beginning is basically used in the case when there is not one attached to a config, which, I believe, is exclusively in the case of tests right now. I'm not a fan of still having the logger.js file either but found it difficult to remove entirely. In the interest of getting this change through, it seemed clearer to keep it for now. Do you think it would be better to rename the imported one and then just name this internal reference as `logger`? Because of the way scoping works for `const`, it might actually work ok to assign `const logger = config.obj('logger')` without any problems.

Reply:
Would it be difficult to attach a logger to the config for the tests so that we can get rid of having two of them in this file? Isn't that what its doing here - https://github.com/bpanel-org/bpanel/pull/161/files#diff-782d73bfba5b95d88102bce2ec86e016R57

Reply:
Here's one example: https://github.com/bpanel-org/bpanel/blob/3a8c95382d3b1a63d027d8a2ec0d4e2f0b14cea2/server/test/configHelpers-test.js#L113

I'll give it one more go over though to see what I can tease out. Let me know if you have any other ideas or see any other places that could be tightened up in the meantime.

Reply:
We just need to be careful about letting on too much tech debt

Reply:
Done!